### PR TITLE
Dev database pull

### PR DESCRIPTION
**Pull request by dfilteau**
IS10 scripts pushed to main repo

### CODE REVIEW

File: .github/workflows/app-logging.yml [hunk 1]
```diff
@@ -2,15 +2,17 @@ name: PIMS Logging CI
 
 on:
```
Comment (Fosol) [inline .github/workflows/app-logging.yml:3]: Seems unrelated to the PR

Comment (Fosol) [reply .github/workflows/app-logging.yml:3]: What's the purpose of the Action?

File: .github/workflows/image-scan-analysis.yml [hunk 2]
```diff
@@ -0,0 +1,133 @@
+name: Security CI
```
Comment (Fosol) [inline .github/workflows/image-scan-analysis.yml:1]: Seems unrelated to the PR

File: .github/workflows/image-scan-analysis.yml [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+      actions: read
+      contents: read
+      security-events: write
+    env:
+      working-directory: ./openshift/4.0/templates/Logging
+      
+      
+    steps:
+    - uses: actions/checkout@v1
+    - name: Build PIMS Logging
```
Comment (Fosol) [inline .github/workflows/image-scan-analysis.yml:114]: Is this related to the Action also in this PR related to logging?

File: database/mssql/scripts/dbscripts/PSP_PIMS_S10_00/Alter Down/01_PSP_PIMS_S10_00_Alter.sql [hunk 4]
```diff
@@ -0,0 +1,9602 @@
+// Script generated by Aqua Data Studio Schema Synchronization for MS SQL Server 2016 on Fri Sep 03 17:12:13 PDT 2021
```
Comment (Fosol) [inline database/mssql/scripts/dbscripts/PSP_PIMS_S10_00/Alter Down/01_PSP_PIMS_S10_00_Alter.sql:1]: I am assuming the **PSP_PIMS_S10_00** folder contains the scripts for Sprint 10?

**Review by asanchezr — APPROVED** [review]

File: database/mssql/scripts/dbscripts/PSP_PIMS_S10_00/Alter Down/01_PSP_PIMS_S10_00_Alter.sql (hunk 4 above)
Comment (asanchezr) [reply database/mssql/scripts/dbscripts/PSP_PIMS_S10_00/Alter Down/01_PSP_PIMS_S10_00_Alter.sql:1]: Yes, Jeremy. The README explains the naming convention

File: .github/workflows/image-scan-analysis.yml (hunk 2 above)
Comment (sookeke) [reply .github/workflows/image-scan-analysis.yml:1]: I added  action to scan images before deployment

File: .github/workflows/app-logging.yml (hunk 1 above)
Comment (sookeke) [reply .github/workflows/app-logging.yml:3]: Just a few changes I made and pass to doug

File: .github/workflows/image-scan-analysis.yml (hunk 3 above)
Comment (sookeke) [reply .github/workflows/image-scan-analysis.yml:114]: Image scan using action before deployment to master/test branch

**Comment by dfilteau** [comment]
My comments in red below.

 

Doug Filteau

Little Mountain Systems Inc.

m: 1.306.201.8531

e:  ***@***.***> ***@***.***

w:  <https://hilltopz.com/> https://hilltopz.com

 

“In our profession, precision and perfection are not a dispensable luxury, but a simple necessity.”

*	Nicklaus Wirth

File: .github/workflows/app-logging.yml (hunk 1 above)
Comment (sookeke) [reply .github/workflows/app-logging.yml:3]: The purpose is to run and test the logging part of the app front and backend. 
https://github.com/bcgov/PSP/tree/dev/openshift/4.0/templates/Logging

This is not now I just made a chanage

**Review by sookeke — APPROVED** [review]
